pkg/internal/testutils/utils.go

```diff
@@ -17,3 +27,80 @@ func CreateConfigValueAndAssertOk(t *testing.T, value any) *prefabProto.ConfigVa
 
 	return val
 }
+
+// Recursively sort JSON objects and arrays by keys and values
+func sortJSONData(data interface{}) interface{} {
```

Comment:
This and below are test helper functions to help comparison across inconsistent JSON serialization.

testify has JSONEq built-in but the readability of failed comparisons isn't ideal, I'd rather just have sorted JSON strings to compare in that case.